cuda_phase-retrieval/test/test_algo.hpp

```diff
@@ -0,0 +1,244 @@
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+#include <cufft.h>
+#include <cuComplex.h>
+#include <curand.h>
+#include <curand_kernel.h>
+#include <cmath>
+#include <cstdio>
+#include <time.h>
+#include <iostream>
+#include <complex>
+#include <string>
+#include <random>
+#if defined _MSC_VER
```

Comment:
How do I increase the C++ standard?

Reply:
In the `CMakeLists.txt` file: https://github.com/ComputationalRadiationPhysics/student_project_python_bindings/blob/cb9995ab4cac02634256690fcc28362724cfefae/cuda_phase-retrieval/CMakeLists.txt#L23

Reply:
I changed it to 17 and now there is pybind error evrywhere

Reply:
Based on another comments, the print image feature will be available in test only. So I think changing this part is not really necessary because it will be removed from the main program

Reply:
I will have a look at the C++ standard problem.

Reply:
Looks like NVCC, pybind11 and C++ 17 does not work together in general. We skip this task.